src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -290,6 +290,9 @@ bool Replayer<I>::get_replay_status(std::string* description,
     m_bytes_per_snapshot);
   root_obj["bytes_per_snapshot"] = round_to_two_places(bytes_per_snapshot);
 
+  root_obj["last_snapshot_seconds"] = m_last_snapshot_seconds;
```

Comment:
Sorry, I think I approved a bit too soon.  Let's rename these because a snapshot can also spend some time in the queue waiting for the sync to be allowed to start and this counter only accounts for the actual sync time.
```suggestion
  root_obj["last_snapshot_sync_seconds"] = m_last_snapshot_sync_seconds;
```